doc/content/getting-started/cli/installing-cli/_index.md

```diff
@@ -8,11 +8,13 @@ This section contains instructions for installing the command-line interface.
 
 <!--more-->
 
+{{% tts %}} CLI is called `ttn-lw-cli`. There is also an enterprise specific CLI called `tti-lw-cli` that is nearly identical, but the enterprise `tti-lw-cli` has additional features for [tenant management]({{< ref "reference/api/tenant" >}}). All other commands are the same. For most users, including Cloud and Community Edition users, `ttn-lw-cli` is sufficient.
```

Comment:
I would rephrase this to something like "There are two versions of TTS CLI: `ttn-lw-cli` and `tti-lw-cli`. For most users `ttn-lw-cli` is sufficient, as it supports a full set of commands to manage all TTS features. For TTS Enterprise, we recommend using `tti-lw-cli`, as this version of the CLI contains additional commands for tenant management, compared to `ttn-lw-cli`."